package isn_t_this_e_not_i.now_waypoint_core.domain.auth.exception.follow;

public class SelfFollowException extends RuntimeException{
    public SelfFollowException(String message) {
        super(message);
    }
}
